packages/sit-onyx/src/components/OnyxSidebar/OnyxSidebar.vue

```diff
@@ -55,11 +64,69 @@ const resizeHandleProps = computed(
       alignment: props.alignment === "left" ? "right" : "left",
     }) as const,
 );
+const { width: windowWidth } = useResizeObserver();
+
+const _isDrawerOpen = ref(false);
+
+const shouldCollapse = computed(() => {
+  if (!props.collapseSidebar) return false;
+  const breakpointWidth =
+    typeof props.collapseSidebar === "number"
+      ? props.collapseSidebar
+      : ONYX_BREAKPOINTS[props.collapseSidebar];
+  return windowWidth.value <= breakpointWidth;
+});
+
+const isDrawerOpen = computed<boolean>({
```

Comment:
Since we removed the drawer component and renamed to prop, should we also rename the ref to something like isModalOpen?